src/routes/(admin)/tests/test-[type]/+page.svelte

```
@@ -257,6 +271,26 @@
+				{#each [{ value: 'all', label: 'All' }, { value: 'true', label: 'Mine' }, { value: 'false', label: 'Shared' }] as option}
+					<button
+						type="button"
+						onclick={() => selectMyTestsFilter(option.value as 'all' | 'true' | 'false')}
+						class="rounded-sm px-2 py-0.5 text-xs font-medium transition-colors
+							{(option.value === 'all' && myTests === null) || myTests === option.value
+							? 'bg-muted text-foreground shadow-sm'
+							: 'text-muted-foreground hover:text-foreground'}"
+					>
+						{option.label}
+					</button>
+				{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)